### PR TITLE
[docs] Reflect new structure of cookie config object since RW 6.4

### PR DESCRIPTION
**Pull request by Philzen**
See #9248

### CODE REVIEW

**Comment by Philzen** [comment]
If these changes are fine, it may also be worth adding them over to the versioned docs folder.

File: docs/docs/auth/dbauth.md [hunk 1]
```diff
@@ -352,11 +357,15 @@ To do this, set the `cookie.Domain` property in your `api/src/functions/auth.js`
...
-  SameSite: 'Strict',
-  Secure: process.env.NODE_ENV !== 'development' ? true : false,
-  Domain: 'example.com'
+  attributes: {
+    HttpOnly: true,
+    Path: '/',
+    SameSite: 'Strict',
+    Secure: process.env.NODE_ENV !== 'development' ? true : false,
+    // highlight-next-line
+    Domain: 'example.com'
```
Comment (Tobbe) [inline docs/docs/auth/dbauth.md:366]: The text above says "set the `cookie.Domain` property". Should that also be updated to say `cookie.attributes.Domain`? Or is it better to just say "set the `Domain` property", and not care exactly how it's nested?

Comment (Philzen) [reply docs/docs/auth/dbauth.md:366]: Good catch :+1:  → 978ad2b6c3b59363fb8fd34852e369ecceb478bb

**Comment by Philzen** [comment]
If it's OK this way now, i can also update the v6 and v7 branches of the docs.

File: docs/docs/auth/dbauth.md [hunk 2]
```diff
@@ -316,14 +316,19 @@ These options determine how the cookie that tracks whether the client is authori
...
+    Path: '/',
+    SameSite: 'Strict',
+    Secure: true,
+    // Domain: 'example.com',
+  },
+  // name: 'session_%port%' 
 }
 ```
 
+As shown above the cookie name defaults to `'session_%port%'` but can also be customized, where `%port% will be replaced with the port the api server is running on.
```
Comment (Tobbe) [inline docs/docs/auth/dbauth.md:330]:
```suggestion
As shown above the cookie name defaults to `'session_%port%'` but can also be customized, where `%port%` will be replaced with the port the api server is running on.
```

**Comment by Tobbe** [comment]
> If it's OK this way now, i can also update the v6 and v7 branches of the docs.

I found one tiny thing that I fixed. So if you get the latest code from the branch, you can use that and copy it over to v6 and v7. Thanks!

**Comment by Philzen** [comment]
@Tobbe All done. Also found a section that we overlooked and updated that as well, see 957d655a56c5dcc1af1495ae3d365790ed273064

File: docs/docs/auth/dbauth.md [hunk 3]
```diff
@@ -348,15 +353,19 @@ See [WebAuthn Configuration](#function-config) section below.
 
 By default, the session cookie will not have the `Domain` property set, which a browser will default to be the [current domain only](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#define_where_cookies_are_sent). If your site is spread across multiple domains (for example, your site is at `example.com` but your api-side is deployed to `api.example.com`) you'll need to explicitly set a Domain so that the cookie is accessible to both.
 
-To do this, set the `cookie.Domain` property in your `api/src/functions/auth.js` configuration, set to the root domain of your site, which will allow it to be read by all subdomains as well. For example:
+To do this, set the `cookie.attibutes.Domain` property in your `api/src/functions/auth.js` configuration, set to the root domain of your site, which will allow it to be read by all subdomains as well. For example:
```
Comment (Tobbe) [inline docs/docs/auth/dbauth.md:356]:
```suggestion
To do this, set the `cookie.attributes.Domain` property in your `api/src/functions/auth.js` configuration, set to the root domain of your site, which will allow it to be read by all subdomains as well. For example:
```

File: docs/versioned_docs/version-6.x/auth/dbauth.md [hunk 4]
```diff
@@ -317,15 +322,19 @@ See [WebAuthn Configuration](#function-config) section below.
 
 By default, the session cookie will not have the `Domain` property set, which a browser will default to be the [current domain only](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#define_where_cookies_are_sent). If your site is spread across multiple domains (for example, your site is at `example.com` but your api-side is deployed to `api.example.com`) you'll need to explicitly set a Domain so that the cookie is accessible to both.
 
-To do this, set the `cookie.Domain` property in your `api/src/functions/auth.js` configuration, set to the root domain of your site, which will allow it to be read by all subdomains as well. For example:
+To do this, set the `cookie.attibutes.Domain` property in your `api/src/functions/auth.js` configuration, set to the root domain of your site, which will allow it to be read by all subdomains as well. For example:
```
Comment (Tobbe) [inline docs/versioned_docs/version-6.x/auth/dbauth.md:325]:
```suggestion
To do this, set the `cookie.attributes.Domain` property in your `api/src/functions/auth.js` configuration, set to the root domain of your site, which will allow it to be read by all subdomains as well. For example:
```

File: docs/versioned_docs/version-7.x/auth/dbauth.md [hunk 5]
```diff
@@ -345,15 +350,19 @@ See [WebAuthn Configuration](#function-config) section below.
 
 By default, the session cookie will not have the `Domain` property set, which a browser will default to be the [current domain only](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#define_where_cookies_are_sent). If your site is spread across multiple domains (for example, your site is at `example.com` but your api-side is deployed to `api.example.com`) you'll need to explicitly set a Domain so that the cookie is accessible to both.
 
-To do this, set the `cookie.Domain` property in your `api/src/functions/auth.js` configuration, set to the root domain of your site, which will allow it to be read by all subdomains as well. For example:
+To do this, set the `cookie.attibutes.Domain` property in your `api/src/functions/auth.js` configuration, set to the root domain of your site, which will allow it to be read by all subdomains as well. For example:
```
Comment (Tobbe) [inline docs/versioned_docs/version-7.x/auth/dbauth.md:353]:
```suggestion
To do this, set the `cookie.attributes.Domain` property in your `api/src/functions/auth.js` configuration, set to the root domain of your site, which will allow it to be read by all subdomains as well. For example:
```

File: docs/versioned_docs/version-8.0/auth/dbauth.md [hunk 6]
```diff
@@ -348,15 +353,19 @@ See [WebAuthn Configuration](#function-config) section below.
 
 By default, the session cookie will not have the `Domain` property set, which a browser will default to be the [current domain only](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#define_where_cookies_are_sent). If your site is spread across multiple domains (for example, your site is at `example.com` but your api-side is deployed to `api.example.com`) you'll need to explicitly set a Domain so that the cookie is accessible to both.
 
-To do this, set the `cookie.Domain` property in your `api/src/functions/auth.js` configuration, set to the root domain of your site, which will allow it to be read by all subdomains as well. For example:
+To do this, set the `cookie.attibutes.Domain` property in your `api/src/functions/auth.js` configuration, set to the root domain of your site, which will allow it to be read by all subdomains as well. For example:
```
Comment (Tobbe) [inline docs/versioned_docs/version-8.0/auth/dbauth.md:356]:
```suggestion
To do this, set the `cookie.attributes.Domain` property in your `api/src/functions/auth.js` configuration, set to the root domain of your site, which will allow it to be read by all subdomains as well. For example:
```

File: docs/versioned_docs/version-8.2/auth/dbauth.md [hunk 7]
```diff
@@ -348,15 +353,19 @@ See [WebAuthn Configuration](#function-config) section below.
 
 By default, the session cookie will not have the `Domain` property set, which a browser will default to be the [current domain only](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#define_where_cookies_are_sent). If your site is spread across multiple domains (for example, your site is at `example.com` but your api-side is deployed to `api.example.com`) you'll need to explicitly set a Domain so that the cookie is accessible to both.
 
-To do this, set the `cookie.Domain` property in your `api/src/functions/auth.js` configuration, set to the root domain of your site, which will allow it to be read by all subdomains as well. For example:
+To do this, set the `cookie.attibutes.Domain` property in your `api/src/functions/auth.js` configuration, set to the root domain of your site, which will allow it to be read by all subdomains as well. For example:
```
Comment (Tobbe) [inline docs/versioned_docs/version-8.2/auth/dbauth.md:356]:
```suggestion
To do this, set the `cookie.attributes.Domain` property in your `api/src/functions/auth.js` configuration, set to the root domain of your site, which will allow it to be read by all subdomains as well. For example:
```

File: docs/versioned_docs/version-8.3/auth/dbauth.md [hunk 8]
```diff
@@ -348,15 +353,19 @@ See [WebAuthn Configuration](#function-config) section below.
 
 By default, the session cookie will not have the `Domain` property set, which a browser will default to be the [current domain only](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#define_where_cookies_are_sent). If your site is spread across multiple domains (for example, your site is at `example.com` but your api-side is deployed to `api.example.com`) you'll need to explicitly set a Domain so that the cookie is accessible to both.
 
-To do this, set the `cookie.Domain` property in your `api/src/functions/auth.js` configuration, set to the root domain of your site, which will allow it to be read by all subdomains as well. For example:
+To do this, set the `cookie.attibutes.Domain` property in your `api/src/functions/auth.js` configuration, set to the root domain of your site, which will allow it to be read by all subdomains as well. For example:
```
Comment (Tobbe) [inline docs/versioned_docs/version-8.3/auth/dbauth.md:356]:
```suggestion
To do this, set the `cookie.attributes.Domain` property in your `api/src/functions/auth.js` configuration, set to the root domain of your site, which will allow it to be read by all subdomains as well. For example:
```

File: docs/versioned_docs/version-8.4/auth/dbauth.md [hunk 9]
```diff
@@ -348,15 +353,19 @@ See [WebAuthn Configuration](#function-config) section below.
 
 By default, the session cookie will not have the `Domain` property set, which a browser will default to be the [current domain only](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#define_where_cookies_are_sent). If your site is spread across multiple domains (for example, your site is at `example.com` but your api-side is deployed to `api.example.com`) you'll need to explicitly set a Domain so that the cookie is accessible to both.
 
-To do this, set the `cookie.Domain` property in your `api/src/functions/auth.js` configuration, set to the root domain of your site, which will allow it to be read by all subdomains as well. For example:
+To do this, set the `cookie.attibutes.Domain` property in your `api/src/functions/auth.js` configuration, set to the root domain of your site, which will allow it to be read by all subdomains as well. For example:
```
Comment (Tobbe) [inline docs/versioned_docs/version-8.4/auth/dbauth.md:356]:
```suggestion
To do this, set the `cookie.attributes.Domain` property in your `api/src/functions/auth.js` configuration, set to the root domain of your site, which will allow it to be read by all subdomains as well. For example:
```

File: docs/versioned_docs/version-8.1/auth/dbauth.md [hunk 10]
```diff
@@ -348,15 +353,19 @@ See [WebAuthn Configuration](#function-config) section below.
 
 By default, the session cookie will not have the `Domain` property set, which a browser will default to be the [current domain only](https://developer.mozilla.org/en-US/docs/Web/HTTP/Cookies#define_where_cookies_are_sent). If your site is spread across multiple domains (for example, your site is at `example.com` but your api-side is deployed to `api.example.com`) you'll need to explicitly set a Domain so that the cookie is accessible to both.
 
-To do this, set the `cookie.Domain` property in your `api/src/functions/auth.js` configuration, set to the root domain of your site, which will allow it to be read by all subdomains as well. For example:
+To do this, set the `cookie.attibutes.Domain` property in your `api/src/functions/auth.js` configuration, set to the root domain of your site, which will allow it to be read by all subdomains as well. For example:
```
Comment (Tobbe) [inline docs/versioned_docs/version-8.1/auth/dbauth.md:356]:
```suggestion
To do this, set the `cookie.attributes.Domain` property in your `api/src/functions/auth.js` configuration, set to the root domain of your site, which will allow it to be read by all subdomains as well. For example:
```

**Review by Tobbe — CHANGES_REQUESTED** [review]

**Review by Tobbe — APPROVED** [review]

**Pull request merged**